### PR TITLE
Version 1.2.0, per CHANGELOG.md update

### PR DESCRIPTION
**Pull request by pond**
- Added `AlchemyFlux::VERSION` and `AlchemyFlux::DATE`.
- Ruby v2.2.7 or later, default to Ruby 2.4.1.
- Maintenance `bundle update`.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,20 @@
-version 1.0
-- removed MessagePack
+# Change log
+
+## v1.2.0
+
+- Added `AlchemyFlux::VERSION` and `AlchemyFlux::DATE`.
+- Ruby v2.2.7 or later, default to Ruby 2.4.1.
```
Comment (mbdietrich) [inline CHANGELOG.md]: What does this line in the changelog mean? Looking at the diff, "Require Ruby v2.2.7 or later"?

Comment (pond) [inline CHANGELOG.md]: Refers to this part of the change set.

https://github.com/LoyaltyNZ/alchemy-flux/pull/12/files#diff-791b78510bbad547296042214d4650b2L14

**Comment by mbdietrich** [comment]
👍

**Comment by grahamjenson** [comment]
👍

**Pull request merged**